CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
 
 * imageRepositoryOverride,imageRegistryOverride and imageTagOverride are now removed from values.yaml. defaultImageRepository, defaultImageRegistry and defaultImageTag values are introduced in helm charts which sets the default registry, repository and tags for the images. Now the registry, repository and tag for a single image can be configured as per the requirement.
 * The OpenShift Templates were removed from the examples and are no longer supported (#5548)
+* Kafka MirrorMaker 1 has been deprecated in Apache Kafka 3.0.0 and will be removed in Apache Kafka 4.0.O.
+  As a result, the `KafkaMirrorMaker` custom resource which is used to deploy Kafka MirrorMaker 1 has been deprecated in Strimzi as well.
+  The `KafkaMirrorMaker` resource will be removed from Strimzi when we adopt Apache Kafka 4.0.
```

Comment:
```suggestion
  The `KafkaMirrorMaker` resource will be removed from Strimzi when we adopt Apache Kafka 4.0.0
```
or .X?
Not sure what it us supposed to say, but we should clarify minor version.